File: chap1/Taxes.c
// Ces programmes sont sous licence CeCILL-B V1.
#include <stdio.h>

int main()
{
   double ht,ttc;
   printf("Quel est le prix hors taxes ?\n");
   scanf("%lf", &ht);
   ttc = ht + ht * 19.6 / 100;
   printf("Le prix toutes taxes comprises est %.2lf\n",ttc);
}

File: chap1/Taxes.js
// Ces programmes sont sous licence CeCILL-B V1.
var ht,ttc;
ISNprintln("Quel est le prix hors taxes ?");
ht=ISNinputFloat();
ttc=ht + ht * 19.6/100;
ISNprint("Le prix toutes taxes comprises est ");
ISNprintln(ttc);


